### PR TITLE
 JAMES-2904 MessageResult::hasAttachment  at Metadtata read level

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Comment by Arsnael** [comment]
```
[ERROR]   FileMailQueueTest.dequeueShouldBeConcurrent » ConditionTimeout Condition with ...
```
test this please

**Comment by chibenwa** [comment]
```
FileMailQueueTest.dequeueShouldBeConcurrent » ConditionTimeout Condition with ...
```

The new unstable

**Comment by chibenwa** [comment]
test this please

**Comment by Arsnael** [comment]
```
[81e301731917f474715cae31edfb4fd01940f77e] [ERROR]   CassandraGetMessagesMethodTest 
[81e301731917f474715cae31edfb4fd01940f77e] Expecting:
[81e301731917f474715cae31edfb4fd01940f77e]  <null>
[81e301731917f474715cae31edfb4fd01940f77e] to be equal to:
[81e301731917f474715cae31edfb4fd01940f77e]  <"part1.37A15C92.A7C3488D@linagora.com">
[81e301731917f474715cae31edfb4fd01940f77e] but was not.
```
smells like related?

**Comment by Arsnael** [comment]
CI restart...

test this please

**Comment by chibenwa** [comment]
test this please

**Comment by Arsnael** [comment]
```
[ERROR]   FileMailQueueTest.dequeueShouldBeConcurrent » ConditionTimeout Condition with ...
```
test this please

**Comment by chibenwa** [comment]
Merged

File: mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageDAO.java [hunk 1]
```diff
@@ -288,16 +289,14 @@ private Property toProperty(UDTValue udtValue) {
         return new Property(udtValue.getString(Properties.NAMESPACE), udtValue.getString(Properties.NAME), udtValue.getString(Properties.VALUE));
     }
 
-    private Stream<MessageAttachmentRepresentation> getAttachments(Row row, FetchType fetchType) {
-        switch (fetchType) {
```
Comment (mbaechler) [inline mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageDAO.java]: is this switch/case removal related  to this commit? I can figure out why

Comment (chibenwa) [reply mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageDAO.java]: MessageAttachmentRepresentation only contains metadata of the attachments, which are now always required to position hasAttachment metadata. Hence this switch case removal.

**Pull request closed without merging**